src/coverlet.msbuild.tasks/MSBuildLogger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using ILogger = Coverlet.Core.Logging.ILogger;
+
+namespace Coverlet.MSbuild.Tasks
+{
+    class MSBuildLogger : ILogger
+    {
+        private readonly TaskLoggingHelper _log;
+
+        public MSBuildLogger(TaskLoggingHelper log) => _log = log;
+
+        public void LogVerbose(string message) => _log.LogMessage(MessageImportance.Low, message);
+
+        public void LogInformation(string message)=> _log.LogMessage(MessageImportance.High, message);
```

Comment:
WIth normal and `v:n` does it show?If so I would use `MessageImportance.Normal`

Reply:
you need to use the dotnet test switch `-v n` for it to show

Reply:
ok but it works also with `MessageImportance.Normal`? I wonder if it's correct use `MessageImportance.High` and not `MessageImportance.Normal`.

Reply:
Unfortunately Normal doesn't show up when using `-v n`.   This may be a bug in dotnet test driver.  I haven't had time to investigate this further.

Reply:
Ok please add comment above like "We use `MessageImportance.High` because with `MessageImportance.Normal` doesn't show anything"